### ISSUE TITLE
Suggestion, small feature

### ISSUE THREAD

**Issue by not-NEKO** [codic12/worm#51]
It would be nice to add a feature in wormc to get the ID of the actual active tag (so you'll be able to use arrow keys to move from a tag to another)

**Comment by codic12** [comment]
I agree, I plan to for the next release implement getters for msot of the properties which have setters, because it's useful for scripts. thanks!
